### PR TITLE
Senior developer job ad

### PR DESCRIPTION
**Pull request by adamantivm**
@patricioreyna could you ptal?

### CODE REVIEW

File: source/jobs/senior.html.erb [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+ - Jobs
+layout: page
+---
+
+<h2>Senior developer for robotics start-up</h2>
+
+<div class="row">
+  <div class="col-sm-8">
+    <p>Ekumen is looking for software developers to participate in developing
```
Comment (patricioreyna) [inline source/jobs/senior.html.erb]: "in developing" --> "in the development of" ?

File: source/jobs/senior.html.erb [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+<h2>Senior developer for robotics start-up</h2>
+
+<div class="row">
+  <div class="col-sm-8">
+    <p>Ekumen is looking for software developers to participate in developing 
+       software for robots using ROS. This can include developing robot 
+navigation, high-level interactions with the supervisor, and business logic. 
+       The project is part of greater efforts conducted jointly with the 
+       creators of the technology doing work that has immediate impact on real
```
Comment (patricioreyna) [inline source/jobs/senior.html.erb]: add a comma ? "...creators of the technology" , "doing work...

File: source/jobs/senior.html.erb [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+    </ul>
+  </div>
+</div>
+
+<div class="row">
+  <div class="col-sm-8">
+    <h3>It’d be nice if you had experience with:</h3>
+
+    <ul>
+      <li>Docker, Jenkins, Git, github, UNIX/Linux, CMake, bash scripting.</li>
```
Comment (patricioreyna) [inline source/jobs/senior.html.erb]: what about adding a "nice to have" or Must (above) about leading experience ? Be it small tech teams, or people and/or 
project management

**Comment by adamantivm** [comment]
@patricioreyna addressed. Ptal again.

**Comment by patricioreyna** [comment]
LGTM thanks

**Pull request merged**